index.html

```diff
@@ -274,5 +277,69 @@ <h2 class="about-me-header">About me</h2>
             </ul>
         </div>
     </section>
+    <!-- contact section-->
+    
+    <section class="contact-section">
+        <h2>
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+        <form
+          action="https://formspree.io/f/mgebdyjo"
+          
+          method="POST"
+          id="contact-form"
+          class="contact-form"
+        >
+          <fieldset>
+            <input
+              class="inputs"
+              type="text"
+              name="full-name"
+              id="fullname"
+              maxlength="30"
+              placeholder="full name"
```

Comment:
- [ ] Please make your first letter should capital because it not looking consistent with other form inputs and make your design looks unprofessional.